7_deploy_osh/roles/suse-dev-patcher/tasks/clone-repos.yml

```diff
@@ -0,0 +1,11 @@
+---
+- name: Clone OpenStack-Helm related code repositories
+  git:
+    repo: "{{ item.value.src }}"
+    dest: "/opt/{{ item.key }}"
+    update: yes
+    force: yes
+    version: "{{ item.value.version }}"
+  with_dict: "{{ osh_repos }}"
```

Comment:
nit: we should probably get away of with_dict, but it's so easy to read that way! :)